scripts/helsinki-utils.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module 'helsinki-utils/scripts/fetch-translations' {
```

Comment:
Did you consider adding the types in a PR towards helsinki-utils?

Reply:
Yup 👍I wrote down an issue to track it: https://github.com/City-of-Helsinki/helsinki-utils-js/issues/5